infra/cluster_fleet_memberships.tf

```diff
@@ -44,5 +50,8 @@ resource "google_gke_hub_membership" "my_fleet_membership_config" {
       resource_link = "//container.googleapis.com/${google_container_cluster.my_cluster_config.id}"
     }
   }
+  depends_on = [
+    module.enable_multi_cluster_google_apis
+  ]
```

Comment:
See APIs needed for Fleet membership in [this cloud.google.com doc](https://cloud.google.com/anthos/fleet-management/docs/before-you-begin#enable_apis).